src/tokens.scss

```diff
@@ -0,0 +1,4 @@
+:host {
+  --ds-auro-skeleton-background-color: var(--ds-color-background-subtle-default, $ds-color-background-subtle-default);
+  --ds-auro-skeleton-background-secondary-color: var(--ds-color-background-primary-300-default, $ds-color-background-primary-300-default);
```

Comment:
These are missing the `#{...}` around the SCSS variable in order to make them compile correctly.

Reply:
I had thought we only need to do this in the `color.scss` file. This has not been done in any other PR.